pymc/distributions/continuous.py

```diff
@@ -2100,6 +2106,12 @@ def dist(cls, beta, *args, **kwargs):
         assert_negative_support(beta, "beta", "HalfCauchy")
         return super().dist([0.0, beta], **kwargs)
 
+    def get_moment(rv, size, loc, beta):
+        mean, _ = at.broadcast_arrays(at.as_tensor(0.0, dtype=rv.dtype), beta)
```

Comment:
Does something like this also work?

```suggestion
        mean = at.zeros_like(beta)
```

Reply:
Is zero a good point to start sampling from this distribution?

Reply:
Looking at V3 I think the median (beta) takes precedence over the mode (0), so we were probably using the former to initialize halfcauchys

Reply:
Oh, I'll use the median instead then. Maybe we should list the order of preference for the moments in #5078